libs/utils/decorate.js

```diff
@@ -77,13 +85,12 @@ export function decorateBlockText(el, config = ['m', 's', 'm'], type = null) {
         decorateIconArea(el);
       }
     }
-    const emptyEls = el.querySelectorAll('p:not([class]), ul:not([class]), ol:not([class])');
+    const bodyStyle = `body-${config[1]}`;
+    const emptyEls = el.querySelectorAll('p:not([class]), ul:not([class]), ol:not([class], div:not([class]))');
```

Comment:
Could we benefit from some `:is` magic?
```suggestion
    const emptyEls = el.querySelectorAll(':is(p, ul, ol, div):not([class])');
```
Anyway, I think there may be a typo here, as `ol:not([class]` doesn't get closed, but is chained to the `div` as well, might deserve another look.